javascript/react/src/verbDescriptions/UnhotDescription.tsx

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import type { UnhotArgs } from '@data-wrangling-components/core'
+import { memo, useMemo } from 'react'
+
+import type { StepDescriptionProps } from '../types.js'
+import { createRowEntries } from '../verbForm/createRowEntries.js'
+import { VerbDescription } from '../verbForm/VerbDescription.js'
+
+export const UnhotDescription: React.FC<StepDescriptionProps<UnhotArgs>> = memo(
+	function UnhotDescription(props) {
```

Comment:
This seems like it is a missing a few params, like the prefix and keep original columns